include/aikido/common/StepSequence.hpp

```diff
@@ -92,11 +92,11 @@ class StepSequence
   std::size_t mNumSteps;
 };
 
-class StepSequence::const_iterator
-    : public boost::iterator_facade<StepSequence::const_iterator,
```

Comment:
One possible additional option to set: `BreakBeforeInheritanceComma: true`. I think that would bring the code back closer to this, although then we'd have commas before all the classes that are being inherited (similar to what it looks like with initializer lists).

Reply:
With `BreakBeforeInheritanceComma: true` it looks better in terms of readability. So should I add go ahead with adding it?

Reply:
Sounds good, let's go for it.